options/locale/locale_en-US.ini

```diff
@@ -3411,3 +3411,20 @@ runs.valid_workflow_helper = Workflow config file is valid.
 runs.invalid_workflow_helper = Workflow config file is invalid. Please check your config file: %s
 
 need_approval_desc = Need approval to run workflows for fork pull request.
+
+[check_runs]
+status.pending = Not started
+status.runing = Still runing, has run %s
+conclusion.action_required = Request more actions after %s
+conclusion.cancelled = Cancelled after %s
+conclusion.failure = Failing after %s
+conclusion.neutral = Neutral in %s
+conclusion.success = Successful in %s
+conclusion.skipped = Has been skipped in %s
+conclusion.stale = Has been stale after %s
+conclusion.timed_out = Has been timeout after %s
+annotation.notice = Notice
+annotation.error = Error
+annotation.failure = Failure
+annotation.warning = Warning
```

Comment:
Would move such simple words to a top-level translation. It makes no sense to having these words translated again and again because every feature has it's own translation scope. We should try to share more such simple translations.